src/components/note/note-test.stories.tsx

```diff
@@ -0,0 +1,86 @@
+import React from "react";
+import { EditorState, ContentState, convertFromHTML } from "draft-js";
+import Note, { NoteProps } from "./note.component";
+import { ActionPopover, ActionPopoverItem } from "../action-popover";
+
+export default {
+  title: "Note/Test",
+  includeStories: "DefaultStory",
+  parameters: {
+    info: { disable: true },
+    chromatic: {
+      disable: true,
+    },
+  },
+};
+
+export const DefaultStory = ({
+  name,
+  createdDate,
+  ...args
+}: Omit<NoteProps, "noteContent">) => {
+  const noteContent = EditorState.createWithContent(
+    ContentState.createFromText("Here is some plain text content")
+  );
+  return (
+    <Note
+      name={name}
+      noteContent={noteContent}
+      createdDate={createdDate}
+      {...args}
+    />
+  );
+};
+
+DefaultStory.story = {
+  name: "default",
+  args: {
+    name: "Lauren Smith",
+    createdDate: "23 May 2020, 12:08 PM",
+  },
+};
+
+export const NoteComponent = ({
+  text,
+  ...props
+}: Omit<NoteProps, "noteContent" | "createdDate"> & { text?: string }) => {
+  const initialValue = text
+    ? EditorState.createWithContent(ContentState.createFromText(text))
+    : EditorState.createEmpty();
+
+  return (
+    <Note
+      title="Here is a Title"
+      name="Lauren Smith"
+      noteContent={initialValue}
+      createdDate="23 May 2020, 12:08 PM"
+      {...props}
+    />
+  );
+};
```

Comment:
```suggestion
  title,
  name,
  createdDate,
  ...rest
}: Omit<NoteProps, "noteContent"> & { text?: string }) => {
  const initialValue = text
    ? EditorState.createWithContent(ContentState.createFromText(text))
    : EditorState.createEmpty();

  return (
    <Note
      title={title || "Here is a Title"}
      name={name || "Lauren Smith"}
      noteContent={initialValue}
      createdDate={createdDate || "23 May 2020, 12:08 PM"}
      {...rest}
    />
  );
};
```

I noticed in a few cypress tests that were passing `Note`'s `createdDate`, `title` and `name` props to the this mock component. This currently works since we are spreading the remaining props via `...props`, but to make it clear to whoever sees this file in the future that these three props have default values that can be overridden, I would amend the component and its type definition slightly to the above.